.github/workflows/release.yaml

```diff
@@ -71,6 +90,166 @@ jobs:
           files: web5_cli-x86_64-unknown-linux-musl
           repo-token: ${{ secrets.GITHUB_TOKEN }}
 
```

Comment:
All the above `build_cli*` jobs are not present in [tbdex-rs release pipeline](https://github.com/TBD54566975/tbdex-rs/blob/main/.github/workflows/release.yml#L15-L23). Is there a reason we need it for web5?

Reply:
The cli only lives in web5-rs (and not tbdex) so we want to release a cli with every version